modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf

```diff
@@ -41,16 +41,16 @@ process GATK4_DETERMINEGERMLINECONTIGPLOIDY {
     ) : ""
     def contig_ploidy = contig_ploidy_table ? "--contig-ploidy-priors ${contig_ploidy_table}" : ""
 
-    def avail_mem = 3
```

Comment:
also no version bump here

Reply:
fixed, the docker image wasn't available at the time